### ISSUE TITLE
Add "Task" to the ISSUE_TEMPLATE.md

### ISSUE THREAD

**Issue by tballison** [apache/incubator-stormcrawler#1411]
What kind of issue is this?

 - [ ] Question. This issue tracker is not the best place for questions. If you want to ask how to do
       something, or to understand why something isn't working the way you expect it to, use StackOverflow
       instead with the label 'stormcrawler': https://stackoverflow.com/questions/tagged/stormcrawler 

 - [ ] Bug report. If you’ve found a bug, please include a test if you can, it makes it a lot easier to fix things. Use the label 'bug' on the issue.
 
 - [ ] Feature request. Please use the label 'wish' on the issue.

It might make sense to add a "Task" option to the issue template for dependency upgrades and other maintenance work that doesn't fall within "bug" or "feature request".

**Comment by tballison** [comment]
I'm opening this for discussion. I kind of want this, but I don't know if fellow devs would find it useful.

**Comment by rzo1** [comment]
Sounds good. Feel free @tballison

**Comment by Brijeshthummar02** [comment]
@rzo1 may i do it, as it's almost 2-3 months after opening the issue??

**Comment by rzo1** [comment]
Feel free

**Comment by Brijeshthummar02** [comment]
yeah sounds good! i was asking do we need to add new yml files for issue_template or let just add sub task in current issue_temp.md file.  

What i feel we can modify it according to other apache orgs as they use like bug-report.yml, sub-task.yml, feature-req.yml, what are your thoughts on it.

**Comment by Brijeshthummar02** [comment]
@rzo1 follow up for above msg.

**Comment by rzo1** [comment]
I don’t have a strong opinion on it. If the files differ significantly in content, I’m fine with adding new ones. Electron has a nice form-based issue submission system—might be worth exploring: https://github.com/electron/electron/issues

**Comment by Brijeshthummar02** [comment]
@rzo1 that sounds valid, i will go through the electron and will add relevant files and changes what I feel it should be there.
